src/device.rs

```diff
@@ -248,9 +248,29 @@ impl DiskDevices {
                 String::from_utf8_lossy(d.file_system()).to_string(),
                 pool_sizes,
             );
-            result
-                .mount_points
-                .push((Path::from(d.mount_point()), index));
+
+            // On macOS APFS disk users' data is mounted in '/System/Volumes/Data'
+            // but fused transparently and presented as part of the root filesystem.
+            // It requires remapping Data volume path for this DiskDevice to '/'.
+            // https://www.swiftforensics.com/2019/10/macos-1015-volumes-firmlink-magic.html
+            // https://eclecticlight.co/2020/01/23/catalina-boot-volumes/
+            if cfg!(target_os = "macos") {
+                if String::from_utf8_lossy(d.file_system()) == "apfs"
+                    && d.mount_point().to_string_lossy() == "/System/Volumes/Data"
```

Comment:
This condition can be flattened and slightly simplified:
```rust
if cfg!(target_os = "macos") 
   && d.file_system() == b"apfs"
   && d.mount_point().to_str() == Some("/System/Volumes/Data") {

}```

and remove the repetition of the else clause.

Reply:
you are right. I have not written any code for long... and also not used github as you can see by small mess I created. But things are coming back quickly.

Doing it now.